plugins/jobtype/jobtypes/gobblin/private.properties

```diff
@@ -0,0 +1,7 @@
+jobtype.class=azkaban.jobtype.connectors.gobblin.GobblinHadoopJob
+
+jobtype.classpath=${plugin.dir}/lib/*
+gobblin.config.preset.dir=${plugin.dir}/presets
+
+#Gobblin file system URI TODO Can we make this cluster independent?
+fs.uri=hdfs://localhost:9000
```

Comment:
Just a quick check: do you want to provide this as the default value?
This could make sense if people want to spin up Azkaban and Gobblin job type on the local machine with a local cluster. 
Or do you think this should be left blank or putting a place holder?